src/components/content-tab-settings.tsx

```diff
@@ -78,7 +79,7 @@ export function ContentTabSettings( { selectedSite }: ContentTabSettingsProps )
 						<div>
 							<span>{ selectedSite.enableHttps ? __( 'Enabled' ) : __( 'Disabled' ) }</span>{ ' ' }
 							{ ! isCertificateTrusted && selectedSite.enableHttps && (
-								<Button variant="link" onClick={ () => getIpcApi().openCertificate() }>
+								<Button variant="link" onClick={ () => getIpcApi().trustCertificate() }>
```

Comment:
```suggestion
								<Button
									variant="link"
									onClick={ async () => {
										await getIpcApi().trustCertificate();
										dispatch( certificateTrustApi.util.invalidateTags( [ 'CertificateTrust' ] ) );
									} }
								>
```
Sorry if you're already working on this and I'm just nagging, @gavande1, but this change (that I mentioned [previously](https://github.com/Automattic/studio/pull/1327#pullrequestreview-2820620472)) is the only thing that I see remaining here.

Reply:
Yes. Was working on it. It seems I had to explicitly call `refetch` to update the UI. Addressed in [8e3e367](https://github.com/Automattic/studio/pull/1327/commits/8e3e367dfa493943ffef9fda8e6eee694c7d3a33).